lib/libzfs/os/freebsd/libzfs_compat.c

```diff
@@ -281,7 +281,7 @@ int
 zpool_nextboot(libzfs_handle_t *hdl, uint64_t pool_guid, uint64_t dev_guid,
     const char *command)
 {
-	zfs_cmd_t zc = { 0 };
+	zfs_cmd_t zc = {"\0"};
```

Comment:
We should be consistent in the way we initialize `zc`. See line 233. Can you make them consistent?

Reply:
Done!